telemetry/definitions/commonDefinitions.json

```
@@ -1694,6 +1749,16 @@
             "type": "string",
             "description": "Generic name of a template"
         },
+        {
+            "name": "testGenerationJobGroupName",
+            "type": "string",
+            "description": "Unit test generation job group name"
+        },
+        {
+            "name": "testGenerationJobId",
```

a group name and job id could be used in many metrics. please do not add hyper-specific fields for every metric.

I already see fields like `codeTransformJobId`, and that's another case where we're paying the price for those fields being over-specific.

```suggestion
            "name": "jobGroup",
            "type": "string",
            "description": "Job group name associated with the metric"
        },
        {
            "name": "jobId",
```

---

Will update with this